data/ecosystems/a/astar-network.toml

```diff
@@ -4,31 +4,45 @@ title = "Astar Network"
 sub_ecosystems = [
   "ACryptoS",
   "adaotop",
+  "AirGap",
   "ArthSwap",
   "Artzero",
   "AstridDao",
+  "Band Protocol",
   "Celer Network cBridge",
+  "Coinversation",
+  "Covalent",
```

Comment:
can be removed since it is compatible with almost all ecos. Hence we consider it part of the "General" crypto eco